lib/new_relic/agent/datastores/metric_helper.rb

```diff
@@ -101,7 +101,9 @@ def self.metrics_from_sql(product, sql)
         end
 
         def self.operation_from_sql(sql)
-          NewRelic::Agent::Database.parse_operation_from_query(sql) || OTHER
+          operation = NewRelic::Agent::Database.parse_operation_from_query(sql)
+          operation = OTHER if operation.eql?('other')
```

Comment:
Would it be possible to return `OTHER` from `parse_operation_from_query`, instead of the string `'other'`? Then maybe we could just call the method and not need to do any checking? could decrease object allocation a little as well

Reply:
Different tests seem to require `'OTHER'`, `'Other'`, or `'other'` which gave me trouble when I was trying to standardize on things.

I suppose it's possible to use constants for all three cases to prevent instantiating more than one copy of any of these 3 strings though, yeah. I'll take a look.

Reply:
Addressed with [2e60b55](https://github.com/newrelic/newrelic-ruby-agent/pull/1145/commits/2e60b55723f1c28d6ac1901b15c6c48fb5b8a889)